pysat/_instrument.py

```diff
@@ -189,6 +193,25 @@ def __init__(self, platform=None, name=None, tag=None, sat_id=None,
                  min_label='value_min', max_label='value_max',
                  fill_label='fill', *arg, **kwargs):
 
+        # Raise warnings about deprecated kwargs
+        dwarns = list()
+        label_kwargs = ['units_label', 'name_label', 'notes_label',
+                        'desc_label', 'min_label', 'max_label', 'fill_label']
+        dwarns.append("".join(["Instrument attributes [",
+                               ", ".join(label_kwargs), "] will be accessible",
+                               " through `Instrument.meta.labels` or ",
```

Comment:
My initial reaction is the guidance should point to only one location, the `Instrument.meta.labels`

Reply:
I made a suggestion to that effect.